developer_manual/core/ocs-share-api.rst

```diff
@@ -129,3 +130,64 @@ Statuscodes:
 * 400 - wrong or no update parameter given
 * 403 - public upload disabled by the admin
 * 404 - couldn't update share
+
+
+Federated Cloud Shares
+======================
```

Comment:
Never mind, it doesn't work across manuals, as each manual is a separate document. ref tags cross-reference arbitrary sections within a single document, like the Setting Strong Permissions section of the Installation Wizard page in the Admin manual:

````
.. _strong_perms:

Setting Strong Directory Permissions
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Then reference the tag like this example on the Command Line Installation page:

````
2. Apply the correction permissions to your ownCloud files and directories (see 
:ref:`strong_perms`.)